src/i18n/locales/zh.js

```diff
@@ -0,0 +1,213 @@
+const chinese = {
+  name: "Simplified Chinese",
+  native_name: "简体中文",
+  code: "zh",
+};
+
+const zh = {
+  translation: {
+    report_bug: "报告问题",
+    import: "导入",
+    file: "文件",
+    new: "新建",
+    new_window: "新窗口",
+    open: "打开",
+    save: "保存",
+    save_as: "另存为",
+    save_as_template: "保存为模板",
+    template_saved: "模板已保存！",
+    rename: "重命名",
+    delete_diagram: "删除图表",
+    are_you_sure_delete_diagram: "确定要删除此图表吗？此操作不可逆转。",
+    oops_smth_went_wrong: "糟糕！出了些问题。",
+    import_diagram: "导入图表",
+    import_from_source: "从源导入",
+    export_as: "导出为",
+    export_source: "导出源",
+    models: "模型",
+    exit: "退出",
+    edit: "编辑",
+    undo: "撤销",
+    redo: "重做",
+    clear: "清除",
+    are_you_sure_clear: "确定要清除图表吗？此操作不可逆转。",
+    cut: "剪切",
+    copy: "复制",
+    paste: "粘贴",
+    duplicate: "复制",
+    delete: "删除",
+    copy_as_image: "复制为图像",
+    view: "视图",
+    header: "标题",
+    sidebar: "侧边栏",
+    issues: "问题",
+    presentation_mode: "演示模式",
+    strict_mode: "严格模式",
+    field_details: "字段详情",
+    reset_view: "重置视图",
+    show_grid: "显示网格",
+    show_cardinality: "显示基数",
+    theme: "主题",
+    light: "明亮",
+    dark: "暗黑",
+    zoom_in: "放大",
+    zoom_out: "缩小",
+    fullscreen: "全屏",
+    settings: "设置",
+    show_timeline: "显示时间轴",
+    autosave: "自动保存",
+    panning: "平移",
+    table_width: "表格宽度",
+    language: "语言",
+    flush_storage: "清除存储",
+    are_you_sure_flush_storage:
+      "确定要清除存储吗？这将不可逆地删除所有您的图表和自定义模板。",
+    storage_flushed: "存储已清除",
+    help: "帮助",
+    shortcuts: "快捷键",
+    ask_on_discord: "在 Discord 上问我们",
+    feedback: "反馈",
+    no_changes: "没有更改",
+    loading: "加载中...",
+    last_saved: "上次保存",
+    saving: "保存中...",
+    failed_to_save: "保存失败",
+    fit_window_reset: "适应窗口/重置",
+    zoom: "缩放",
+    add_table: "添加表",
+    add_area: "添加区域",
+    add_note: "添加注释",
+    add_type: "添加类型",
+    to_do: "待办事项",
+    tables: "表",
+    relationships: "关系",
+    subject_areas: "主题区域",
+    notes: "注释",
+    types: "类型",
+    search: "搜索...",
+    no_tables: "没有表",
```

Comment:
```suggestion
    no_tables: "空空如也",
```

"空空如也" is an idiom meaning completely empty, devoid of anything.

You can change the empty prompts for "relationship," "area," "notes," etc. to this.